.azure/pipelines/ci.yml

```diff
@@ -246,6 +246,7 @@ stages:
       jobName: Windows_arm_build
       jobDisplayName: "Build: Windows ARM"
       agentOs: Windows
+      buildDirectory: $(System.DefaultWorkingDirectory)/eng/
```

Comment:
Suggest changing the default `buildDirectory` value in [default-build.yml](https://github.com/dotnet/aspnetcore/blob/main/.azure/pipelines/jobs/default-build.yml) instead of duplicating this all over

Reply:
Good idea! Done.